.golangci.yml

```diff
@@ -4,6 +4,19 @@ run:
   # while if uncached!
   timeout: 10m
 
+linters:
+  enable:
```

Comment:
This seems like a short list. How did we decide which ones to enable?

Wondering if we should try to enable the same as in CR (potentially split up in multiple PRs if there would be a lot of findings)

Reply:
Thanks @sbueringer 

In my opinion it should be possible to enable all linters. 

Under the previous default settings, `.golangci.yaml` only enabled `[errcheck gosimple govet ineffassign staticcheck unused] ` linter.
![image](https://github.com/user-attachments/assets/d981b086-6e9e-4b57-9fe0-e27c6893b010)

I'll add other linters for next PR.

Reply:
Sounds good, thx!

> In my opinion it should be possible to enable all linters.

Just to avoid misunderstandings. I would only enable the ones enabled in controller-runtime (vs. all :))

Reply:
> Just to avoid misunderstandings. I would only enable the ones enabled in controller-runtime (vs. all :))

Thanks!
I got it.